### PR TITLE
docs(configuration): add a missing option

### PR DESCRIPTION
**Pull request by g-plane**
The `output.globalObject` is useful for UMD build, which should be documented.

Ref: https://github.com/webpack/webpack/issues/6522

### CODE REVIEW

File: src/content/configuration/output.md [hunk 1]
```diff
@@ -340,6 +341,11 @@ If using a function for this option, the function will be passed an object conta
 T> When using the [`ExtractTextWebpackPlugin`](/plugins/extract-text-webpack-plugin), use `[contenthash]` to obtain a hash of the extracted file (neither `[hash]` nor `[chunkhash]` work).
 
 
+## `output.globalObject`
+
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md]:
```suggestion
`string: 'window'`

```

File: src/content/configuration/output.md [hunk 2]
```diff
@@ -340,6 +341,11 @@ If using a function for this option, the function will be passed an object conta
...
+When targeting a library, especially the `libraryTarget` is `'umd'`, this option indicates what global object will be used to mount the library. Default value is `'window'`. If you want to make your UMD build available on both browsers and Node.js, you should set this option to `'this'`.
```
Comment (EugeneHlushko) [inline src/content/configuration/output.md]:
```suggestion
When targeting a library, especially the `libraryTarget` is `'umd'`, this option indicates what global object will be used to mount the library. To make UMD build available on both browsers and Node.js, set `output.globalObject` option to `'this'`.
```

**Comment by g-plane** [comment]
@EugeneHlushko What's wrong with the Travis job?

**Comment by EugeneHlushko** [comment]
@Munter can you check why svg throw content type mismatch?

**Comment by Munter** [comment]
@EugeneHlushko 	It's an undesirable side effect of a strict matching of content types based on what type of relations are pointing at an asset. I think a newer version of assetgraph (hyperlink dependency) has an improvement here.

I'm marking this specific error as TODO in the tests for now so they should pass until I can update hyperlink

**Comment by EugeneHlushko** [comment]
Got it, thank you @Munter 
I've also resolved the conflict and updated the branch

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**